lib/shopify_cli/theme/syncer.rb

```diff
@@ -255,6 +255,11 @@ def update(file)
 
       def ignore?(operation)
         path = operation.file_path
+        ignore_path?(path)
+      end
+
+      def ignore_path?(file_or_path)
+        path = file_or_path.respond_to?(:path) ? file_or_path.path : file_or_path
```

Comment:
I wonder if we could have two methods like `ignore_path?(path)` and `ignore_file?(file)`.

It would allow us to remove a condition and the reflection method. What do you think about it? :)

Reply:
@karreiro good call on this (I updated `ignore?` to `ignore_operation?` to match)... I think this is cleaner now.